docs/source/markdown/podman-logout.1.md

```diff
@@ -21,6 +21,10 @@ All the cached credentials can be removed by setting the **all** flag.
 
 ## OPTIONS
 
+#### **--accept-repositories**
+
+Allow namespaces or repositories rather than just registries.
```

Comment:
Can you give an example directly here what that means?  I think that many users will have troubles imagining the semantics.

Reply:
Yep, updated the docs to provide an example.